bindings/pyroot/pythonizations/python/ROOT/_pythonization/_function_utils.py

```diff
@@ -0,0 +1,230 @@
+from .._numbadeclare import _NumbaDeclareDecorator
+
+try:
+    import re
```

Comment:
`re` is a builtin Python package, you don't need to protect its import (should always succeed).